### DISCUSSION TITLE
Should we call help?

### DISCUSSION THREAD

**Discussion by Sectonid** [Sectonidse/SourceDEFUN#28] [general]
@maksw2 Should we do it? Because i don't think i would be able to fix the Linux's compilation as every time i fix an error a new one appears (completely unrelated to the one that i fixed)

**Comment by maksw2** [comment]
I think we should scrap it and modify enhanced source.

**Reply by Sectonid** [reply]
I'm taking a look if Alien Swarm SDK can compile and run on Linux at all. If it doesn't, then we have to resort Source SDK 2013 (oh no). If it does, then we'll try to integrate CMake into Enhanced Source

**Reply by maksw2** [reply]
I could get deferred swarm to compile on windows. But on here i can only get shaderlib.

**Reply by maksw2** [reply]
i believe it would be easier to move our changes to enhanced source.

**Reply by Sectonid** [reply]
Found out that the Alien Swarm itself does not support Linux nor even macOS.
![image](https://github.com/user-attachments/assets/a4951a07-b17f-402b-a9d1-356be91c063c) ^(there's only Windows icon)

Using Proton and Wine is not a perfect solution. I'm sorry, but we have to use Source SDK 2013. Don't worry though, there's the deferred lighting implementation for it, and we can start with Mapbase (it has a lot of useful entities).
The reason i'm moving SourceDEFUN into Source SDK 2013 is not because i can't play it on Linux, but because everyone on Linux won't be able to play it. There's only few Source mods that support Linux, but thousands of them don't. Having cross-compile is nice especially when Windows users are moving to Linux (i am one of them too but that was a year ago)

**Comment by maksw2** [comment]
windows actions are looking good.
